.github/workflows/ci.yml

```diff
@@ -68,3 +68,5 @@ jobs:
         run: sudo apt install -y dotnet6
       - name: Run script to make sure all files end with EOL 
         run: dotnet fsi eofConvention.fsx
+      - name: prettier
+        run: npx prettier --check --tab-width=4 './**/*.ts'
```

Comment:
isn't there a way to specify tab width via .editorconfig file?

Reply:
BTW this is very brittle because it will always use the last prettier version; we should pin to the current last version now to avoid CI break on us everytime there's a new version that changes the style

Reply:
Done